db/migrate/20181001225906_create_sail_settings.rb

```diff
@@ -0,0 +1,12 @@
+class CreateSailSettings < ActiveRecord::Migration[5.1]
+  def change
+    create_table :sail_settings do |t|
+      t.string :name, null: false
+      t.text :description
+      t.string :value, null: false
+      t.integer :cast_type, null: false, limit: 1
```

Comment:
Use `limit: 2` cause why not?
```suggestion
      t.integer :cast_type, null: false, limit: 2
```